### DISCUSSION TITLE
Time out issue

### DISCUSSION THREAD

**Discussion by enceya** [riffnshred/nhl-led-scoreboard#279] [General]
Hello all, 

I'm fairly new to this but the setup has been pretty straightforward so far. The only issue I've been having is the Pi seems to time out after about 30 minutes. It still has power but the LED board goes black and ssh disconnects. I have to unplug the Pi and reboot it for it to come back on. Has anyone else had this issue? I haven't seen it mentioned. 

Thank you to those who have contributed to this project. It's been great.

**Comment by riffnshred** [comment]
That is on the raspberry pi side of things. It could be happening for many different reasons. I would suggest to try with a different SD card maybe.

**Reply by enceya** [reply]
Ah, Okay. That makes sense. Thank you for your help.
